.github/workflows/release-to-npm.yml

```diff
@@ -17,6 +17,9 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.BOT_ACCESS_TOKEN }}
```

Comment:
We need another token to allow the directly push for main branch since it is protected.
You can trust yourself or create another account as bot.
<img width="886" alt="Screenshot 2024-04-23 at 00 33 38" src="https://github.com/mystist/react-chatgpt/assets/5009454/70757ee2-3a66-4bcd-b8f1-03d8e89922be">


And create a github token under developer settings, add it to the repo secret with name `BOT_ACCESS_TOKEN`
![Uploading Screenshot 2024-04-23 at 00.34.46.png…]()